### PR TITLE
chore: sign v0.4.0 and v0.4.1 by m1sterc001guy

### PR DESCRIPTION
**Pull request by m1sterc001guy**
Probably sufficient to just have one signature, but I want to find an excuse to use my Yubikey :)

### CODE REVIEW

**Review by oleonardolima — APPROVED** [review]
ACK 0f7f1fe82ffaab66dea807fa67decb52332be481

**Review by dpc — DISMISSED** [review]

**Comment by m1sterc001guy** [comment]
Needed to rename my signature file to avoid conflict

**Comment by dpc** [comment]
See https://github.com/fedimint/fedimint/pull/5812

File: releases/v0.4.0-x86_64-linux.SHA256SUMS [hunk 1]
```diff
@@ -3,8 +3,8 @@ ca9e4e57c5766d7fe4f69616af4a7916d498f748bea0d846c443a7a343730b2c  v0.4.0-x86_64-
...
-5dda65e3260e22ed761969bd79dc4407375588303b366de0587d9857b5dff3c7  v0.4.0-x86_64-linux/nixos/fedimint-cli
-2c3d4086ee05c8ce72ee36e036513244ce802d326ce22679a94ece4a92dd33f3  v0.4.0-x86_64-linux/nixos/fedimint-dbtool
-2ac5afee3e642f012efff407a903487dd69aabc6bf356128693753693ef4b314  v0.4.0-x86_64-linux/nixos/fedimintd
-4f565137ef3e9b79cbb99c031dc8b57d70d233a5ef263e247df938d42f379bc8  v0.4.0-x86_64-linux/nixos/gateway-cli
-94afe7e2a47fb242397287abbed84767cadf59094108db747b4caf9b0f991ecc  v0.4.0-x86_64-linux/nixos/gatewayd
+ae499530bd08653327c0fd2f4293c017e4df3d12a4c62dfd75bf35bdc98c4d9d  v0.4.0-x86_64-linux/nixos/fedimint-cli
+8b359a816a8d866ed35caaa286b05bb63581a0ad8554374454df1959aaf95d56  v0.4.0-x86_64-linux/nixos/fedimint-dbtool
+0eed8de41a7243e66201fc29b1a66a92095bf672002be4025bb507aae9172301  v0.4.0-x86_64-linux/nixos/fedimintd
+4eb2ed656d7793832d6df2c7b33febabd58a49bb6fe90ca78a9f66ae27d65b09  v0.4.0-x86_64-linux/nixos/gateway-cli
+04c6a7446655eeaf682f1c9c09e9210d8de4ae01e9e1da5df00c43aa5c79f0d7  v0.4.0-x86_64-linux/nixos/gatewayd
```
Comment (elsirion) [inline releases/v0.4.0-x86_64-linux.SHA256SUMS:10]: Should these change? I don't think so.

Comment (m1sterc001guy) [reply releases/v0.4.0-x86_64-linux.SHA256SUMS]: Fixed!

**Review by elsirion — APPROVED** [review]

**Comment by elsirion** [comment]
Oh, you queued mine already, so I guess you will need to rebase unfortunately :(

**Review by tvolk131 — APPROVED** [review]

**Pull request merged**